app/routes/agenda-items.ts

```diff
@@ -146,11 +146,13 @@ export default class AgendaItemsRoute extends Route {
     this.plannedStartMin = params.plannedStartMin || "";
     this.plannedStartMax = params.plannedStartMax || "";
 
+    /*
```

Comment:
**suggestion:** I guess these lines can be removed

Reply:
Done!